sdk/include/psdk/ntsecapi.h

```diff
@@ -226,7 +226,8 @@ typedef enum _MSV1_0_PROTOCOL_MESSAGE_TYPE {
   MsV1_0CacheLogon,
   MsV1_0SubAuth,
   MsV1_0DeriveCredential,
-  MsV1_0CacheLookup
+  MsV1_0CacheLookup,
+  MsV1_0SetProcessOption
```

Comment:
Should guard this with `_WIN32_WINNT >= _WIN32_WINNT_WINXP` according to latest MS headers.

Reply:
fixed